code/game/machinery/bots/bots.dm

```diff
@@ -55,9 +55,10 @@
 	if(locked)
 		locked = 0
 		emagged = 1
-		to_chat(user, "<span class='warning'>You bypass [src]'s controls.</span>")
+		to_chat(user, "<span class='warning'>You remove [src]'s control restrictions. Swiping again will cause [src] to malfunction.</span>")
```

Comment:
I tried making it as obvious as I could without being immersion breaking, want me make it more obvious?